docs/content/components/accordion-menu.md

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
Just in case you haven't used it -  there is a script named transfer_doc_files.js located in the scripts/ folder that automatically copies the needed files in the doc folders.

Reply:
I use `npm run build:documentation` which from the build.js file I can see that it uses the transfer_doc_files.js, should I run it directly instead?